src/plugins/profiling/server/routes/search_flamechart.ts

```diff
@@ -271,6 +274,7 @@ async function queryFlameGraph(
     logger.info('unique downsampled stacktraces: ' + stackTraceEvents.size);
   }
 
+  logger.info("stackTraceEvents.keys() is " + [...stackTraceEvents.keys()])
```

Comment:
Do we want to expand all the stacktraces' IDs in the log message? We usually have tens of thousands, how are we going to use them?